PWG/Tools/AliYAMLConfiguration.cxx

```diff
@@ -110,7 +111,7 @@ bool AliYAMLConfiguration::AddConfiguration(const YAML::Node node, std::string c
  *
  * @return True if write was successful.
  */
-bool AliYAMLConfiguration::WriteConfiguration(const int index, const std::string filename) const
+bool AliYAMLConfiguration::WriteConfiguration(const std::string filename, const int index) const
```

Comment:
const std::string & ? (until at some point we would finally be allowed to use std::string_view) Would suppress a copy in case a std::string is passed as argument. Also in other cases.